docs/user/commandline_tools.rst

```diff
@@ -299,41 +299,75 @@ Shape files may also be ingested::
 
     $ geomesa ingest -u username -p password -c test_catalog -f shapeFileFeatureName /some/path/to/file.shp
 
-Enabling S3 Ingest
+**Enabling S3 Ingest**
 
-Hadoop ships with an implementation of an S3 Native filesystem denoted by the prefix ``s3n://`` that can be enabled in
-the Hadoop configuration used with GeoMesa Tools. Edit the core-site.xml file in your Hadoop installation. These
-instructions apply to Hadoop 2.5.0 and higher. Note that you must have the environmental variable
-``HADOOP_MAPRED_HOME`` set properly in your environment.
+Hadoop ships with an implementation of a S3 filesystems that can be enabled in the Hadoop configuration used with GeoMesa Tools. GeoMesa Tools can perform ingest using both the second-generation (`s3n`) and third-generation (`s3a`) filesystems. Edit the `$HADOOP_CONF_DIR/core-site.xml` file in your Hadoop installation, as shown below. These instructions apply to Hadoop 2.5.0 and higher. Note that you must have the environment variable ``HADOOP_MAPRED_HOME`` set properly in your environment. Some configurations can substitute ``HADOOP_PREFIX`` in the classpath values below.
 
-    <!-- core-site.xml -->
+Warning: AWS credentials are valueable. They pay for services and control read and write protection for data. If you are running GeoMesa on AWS EC2 instances, it is recommended to use s3a. With s3a, you can omit the Access Key Id and Secret Access keys from `core-site.xml` and rely on IAM roles.:
+
+s3a::
+
+  <!-- core-site.xml -->
     <property>
-        <name>fs.s3n.impl</name>
-        <value>org.apache.hadoop.fs.s3native.NativeS3FileSystem</value>
-        <description>Tell hadoop which class to use to access s3 URLs. This change became necessary in hadoop 2.6.0</description>
-        </property>
+        <name>mapreduce.application.classpath</name>
+        <value>$HADOOP_MAPRED_HOME/share/hadoop/mapreduce/*:$HADOOP_MAPRED_HOME/share/hadoop/mapreduce/lib/*:$HADOOP_MAPRED_HOME/share/hadoop/tools/lib/*</value>
+        <description>The classpath specifically for mapreduce jobs. This override is neeeded so that s3 URLs work on hadoop 2.6.0+</description>
+    </property>
 
+    <!-- OMIT these keys if running on AWS EC2; use IAM roles instead -->
+    <!-- These are Hadoop 2.7 style properties -->
+    <property>
+        <name>fs.s3a.awsAccessKeyId</name>
```

Comment:
@vpipkt I wasn't able to get these working in 2.7 I had to use the 2.6 style properties...I think that is what the documentation for 2.7 says as well:
https://github.com/apache/hadoop/blob/release-2.7.1/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md#authentication-properties-2